### ISSUE TITLE
How to use GPU.js with nvidia-docker?

### ISSUE THREAD

**Issue by JasonShin** [gpujs/gpu.js#341]
Hi, It's a question rather than an issue...

I'd like to know if using GPU.js in nvidia-docker was attempted before =)

References:
- https://github.com/NVIDIA/nvidia-docker

**Comment by robertleeplummerjr** [comment]
I don't think it has, if it did it would fallback to cpu at this time until we get opencl running.
